addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -1,3 +1,95 @@
+// ======= VARIABLES =======
+$o-wblog-read-width: 620px;
+$o-wblog-read-container-sidebar-width :900px;
+$o-wblog-loader-size: 50px;
+
+$o-wblog-title-sizes-variants: (
```

Comment:
Are those really needed? Can't you achieve the same result by applying the right bootstrap classes on the elements directly?

Reply:
> Can't you achieve the same result by applying the right bootstrap classes on the elements directly?

Not really unfortunately :/
I made a try conditionally applying `.small` classes on `display-x` children if some circumstances were occurring , but I end up with tons of qWeb conditions and the visual result was raw.

Moreover we don't have **font-size** progressive classes (like margins and padding), and the bootstrap gap between `display-X` and `display-Y` in  is too huge to handle this.

I think that this SCSS implementation is not bad at the end.  We have more SCSS lines, but the system relies on bootstrap default variables so it will adapt to themes.

Also, let's assume that for some reasons you decided that your theme needs  "100em `.display-1`" titles.  If it's ok for your website but it's too big for the blog cover, you can act on the blogs variables only, keeping the rest of the website intact.